core-services/prow/02_config/generate-boskos.py

```diff
@@ -47,7 +47,13 @@
         'us-east-1': 8,
         'us-east-2': 8,
         'us-west-2': 8
-    }
+    },
+    'aws-2-quota-slice': {
+        'us-east-1': 35,
```

Comment:
What is the basis for the number 35 here?

Reply:
Just guessing.